tsl/src/chunk_api.c

```diff
@@ -1672,3 +1670,24 @@ chunk_create_empty_table(PG_FUNCTION_ARGS)
 
 	PG_RETURN_BOOL(true);
 }
+
+#define CREATE_CHUNK_TABLE_NAME "create_chunk_table"
+
+void
+chunk_api_call_create_empty_chunk_table(const Hypertable *ht, const Chunk *chunk,
+										const char *node_name)
+{
+	const char *create_cmd =
+		psprintf("SELECT %s.%s($1, $2, $3, $4)", INTERNAL_SCHEMA_NAME, CREATE_CHUNK_TABLE_NAME);
+	const char *params[4] = { quote_qualified_identifier(NameStr(ht->fd.schema_name),
+														 NameStr(ht->fd.table_name)),
+							  chunk_api_dimension_slices_json(chunk, ht),
+							  NameStr(chunk->fd.schema_name),
+							  NameStr(chunk->fd.table_name) };
+
+	ts_dist_cmd_close_response(
```

Comment:
I think there's a macro for this `ts_dist_cmd_run_on_data_node` or similar.

Reply:
This macro doesn't with parameters.